composer.json

```diff
@@ -12,6 +12,7 @@
         }
     ],
     "require": {
+        "php": ">=7.3",
```

Comment:
we still support 5.* with this, so i don't want to make this a requirement

Reply:
Just to elaborate: It could be beneficial for future development that PHP5.6 is supported in a branch, i.e. `2.x` and make a decision that the next branch, i.e. `3.x` will support `>=7.4`. It would make the code in `3.x` much more robust and it will also entice users of PHP5.6 to upgrade

Reply:
yea, that's a good point, we can do a version bump to no longer support 5.6 when more robust typing is added

Reply:
Sounds great